File: Data Structures & Algorithms/two-integer-sum/submission-6.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        std::vector<int> result;
        int target_value;
        for (int i = 0; i < nums.size(); i++){
            target_value = target - nums[i];
            if (std::find(nums.begin() + 1 + i, nums.end(), target_value)!=nums.end()){
                auto iterator = std::find(nums.begin() + 1 + i, nums.end(), target_value);
                result.push_back(i);
                result.push_back(std::distance(nums.begin(), iterator));
                return result;
            }
        }
        return result;
    }
};

File: Data Structures & Algorithms/max-consecutive-ones/submission-6.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max = 0
        counts = {}
        for x in nums:
            if x == 1:
                counts[1] = counts.get(1,0) + 1
                if counts[1] > max:
                    max = counts[1]
            else:
                counts[1] = 0;
        return max

        